wrapper/src/main/java/software/amazon/jdbc/wrapper/ConnectionWrapper.java

```diff
@@ -126,9 +130,6 @@ protected void init(final Properties props) throws SQLException {
 
   public void releaseResources() {
     this.pluginManager.releaseResources();
-    if (this.pluginService instanceof CanReleaseResources) {
```

Comment:
Should we clean up pluginManager similar way?

Reply:
we would have to pass pluginManager into LazyCleaner, but sure